packages/babel-helper-wrap-function/src/index.ts

```diff
@@ -123,7 +122,11 @@ function plainFunction(path: NodePath<any>, callId: any, noNewArrows: boolean) {
       });
     }
 
-    if (!retFunction || retFunction.id || node.params.length) {
+    if (
+      !retFunction ||
+      retFunction.id ||
```

Comment:
Probably we can do it by adding `|| isFunctionDeclaration(retFunction)` here.

Reply:
We don't need to change it.
default export use a different template builder.

The wrapper is in another function.
You can check the test case.

Reply:
Oh awesome, thanks for the test!